bioio_base/standard_metadata.py

```diff
@@ -103,3 +126,104 @@ def to_dict(self) -> dict:
             self.FIELD_LABELS[field]: getattr(self, field)
             for field in self.FIELD_LABELS
         }
+
+
+# Helper functions for OME metadata extraction
+
+
+def get_metadata_element(ome_metadata: OME, path: str) -> Optional[_Element]:
+    """
+    Gets the first occurrence of the given XML path if one exists.
+    """
+    xml = XML(ome_metadata.to_xml())
+    return xml.find(path, OME_NS)
+
+
+def binning(ome_metadata: OME) -> Optional[str]:
+    """
+    Extracts the binning setting from the OME metadata.
+    Returns
+    -------
+    Optional[str]
+        The binning setting as a string. Returns None if not found.
+    """
+    try:
+        el = get_metadata_element(
+            ome_metadata, "./Image/Pixels/Channel/DetectorSettings"
+        )
+        if el is not None:
+            return el.get("Binning", None)
+    except Exception as exc:
+        log.warning("Failed to extract Binning setting: %s", exc, exc_info=True)
+    return None
+
+
+def imaged_by(ome_metadata: OME) -> Optional[str]:
+    """
+    Extracts the name of the experimenter (user who imaged the sample).
+    Returns
+    -------
+    Optional[str]
+        The username of the experimenter. Returns None if not found.
+    """
+    try:
+        el = get_metadata_element(ome_metadata, "./Experimenter")
+        if el is not None:
+            return el.get("UserName", None)
+    except Exception as exc:
+        log.warning("Failed to extract Imaged By: %s", exc, exc_info=True)
+    return None
+
+
+def imaging_date(ome_metadata: OME) -> Optional[str]:
+    """
+    Extracts the acquisition date from the OME metadata.
+    Returns
+    -------
+    Optional[str]
+        The acquisition date in ISO format (YYYY-MM-DD) adjusted to Pacific Time.
+        Returns None if the acquisition date is not found or cannot be parsed.
+    """
+    try:
+        el = get_metadata_element(ome_metadata, "./Image/AcquisitionDate")
+        if el is not None and el.text:
+            utc_time = datetime.fromisoformat(el.text.replace("Z", "+00:00"))
+            pacific_time = utc_time.astimezone(ZoneInfo("America/Los_Angeles"))
```

Comment:
Hmm why this step? Can we grab the timezone of the computer as a default instead to be less specific?
```
utc_time = datetime.fromisoformat(el.text.replace("Z", "+00:00"))
local_time = utc_time.astimezone(datetime.datetime.now().astimezone().tzinfo)
return local_time.date().isoformat()
```

Reply:
@pgarrison whatcha think?

Reply:
Is there a pressing need to switch out of Z time?  If there is, Sean's localization makes sense, although does add complexity (to figure out where we are running)